doc/source/reference/arrays.datetime.rst

```diff
@@ -54,20 +54,25 @@ letters, for a "Not A Time" value.
 
     NAT (not a time):
 
-    >>> numpy.datetime64('nat')
+    >>> np.datetime64('nat')
     numpy.datetime64('NaT')
 
 When creating an array of datetimes from a string, it is still possible
 to automatically select the unit from the inputs, by using the
 datetime type with generic units.
 
+For backwards compatibility, datetime64 still parses timezone offsets,
+which it handles by converting to UTC. However, the resulting datetime
+is timezone naive.
```

Comment:
I think we should put this paragraph next to an example, the text alone isn't clear enough to be.

Reply:
The behaviour is already flagged as deprecated, I might have dropped this part actually.